config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  # post "/api/v1/items", to: 'api/v1/items#create'
```

Comment:
Routes here look good -- probably should delete the commented-out ones.

Reply:
that was more for my own learning.